_includes/articles/river-descriptors-psbt.md

```diff
@@ -0,0 +1,56 @@
+{:.post-meta}
+*by [Philip Glazman][], Engineer at [River Financial][]*
+
+*River Financial​ is a challenger financial institution specializing in Bitcoin
+financial services. River Financial’s flagship product, a Bitcoin brokerage,
+provides retail investors with a high-touch platform to buy and sell Bitcoin.*
+
+River Financial leverages two technologies in its wallet software: [Partially
+Signed Bitcoin Transactions (PSBTs)][topic psbt] and [Output Script
+Descriptors][topic descriptors]. The decision to incorporate these standards has
+introduced greater flexibility and interoperability in River’s wallet operations.
+
+The decision to bring PSBTs into the wallet stack was influenced by the concept
+of treating key signers as interfaces. PSBT is a format for signers described in
+[BIP-174][] authored by Andrew Chow. Before this standard, there had been no
```

Comment:
Nit: we typically use the format `BIP174` (but outside authors using a different format is ok with me).  If the report uses our normal format, you can replace the link definition at the bottom with `{% include references.md %}` to get our usual automatic linking for BIPs.